docs/auth/jwt_auth.rst

```diff
@@ -0,0 +1,241 @@
+Pyramid JWT Authentication Example
+==================================
+
+Introduction
+############
+`JWT <https://jwt.io/>`_ is a standard of token, mainly used for securing web API's. 
+This small tutorial aims to explain why a JWT is useful and how we can use it with 
+pyramid to create a secure authentication system for an API server, couples with 
+pyramid's built in `Security Policy 
```

Comment:
It's not a proper noun. If this were phrased such as "see the chapter Security Policy in Pyramid's documentation", then it would be Title Cased.

Also you need to use the link only once, not for each instance. I'll leave the rest for you to remove the link.

```suggestion
pyramid's built in `security policy 
```